README.md

```diff
@@ -0,0 +1,84 @@
+# CNAB to OCI
+
+The intent of CNAB to OCI is to provide a translation tool in between the two formats 
+
+## Getting Started
+
+To get and build the project:
+```sh
+$ go get github.com/docker/cnab-to-oci
+$ cd $GOPATH/src/github.com/docker/cnab-to-oci
+$ make
+```
+By now you should have a binary of the project in the `bin` folder. To run it, execute:
+```sh
+$ bin/cnab-to-oci --help
+```
+
+### Prerequisites
+
+```
+- Make
+- Golang 1.9
+- Git
+```
+
+### Installing
+
+For installing, make sure your `$GOPATH/bin` makes part of the `$PATH`
+
+```sh
+$ make install
+```
+
+This will build and install `cnab-to-oci` into `$GOPATH/bin`
+
+### Usage
+This section is reserved to the usage of `cnab-to-oci` as a translation (binary) tool
+
+#### Pull
+`$ bin/cnab-to-oci pull <docker-ref>`
+
+#### Push
+`$ bin/cnab-to-oci push <bundle file>`
+
+#### Fixup
+The fixup target is meant to attach an image to a digest instead of a tag that can move from time to time.
```

Comment:
The right word is `command` and not `target`, it's a `make` concept.

Reply:
I think it can be re-worded: "The fixup command resolves all the digested reference from a registry and patches the `bundle.json` with them".

Reply:
Cool I prefer your rephrasing too. Gonna change to:

"The fixup command resolves all the digest references from a registry and patches the bundle.json with them."